src/components/LayoutPanels/library/dom/dragToResizePanelWidth.ts

```diff
@@ -1,5 +1,7 @@
 import { convertCssWidthPropertyValueToPixels } from "./convertCssWidthPropertyToPixels";
 
+const AUTO_CLOSE_PANEL_DRAG_THRESHOLD = 25; // px
```

Comment:
Instead of auto closing at the min width, well use 25px for now (Al can readjust)